doc/docker.md

```diff
@@ -202,30 +194,36 @@ started.
 
 Remove/force the icubam containers
 ```
-docker rm -f icubam-server icubam-sms-server icubam-bo-server
+docker rm -f icubam_www_server icubam_sms_server icubam_bo_server
 ```
 
 Get the logs of the server
 ```
-docker logs icubam-server
+docker logs icubam_www_server
```

Comment:
Ok, and I guess this also works fine for the other instances as well? (their logs are also useful)

Reply:
Yes. this is a standard docker command.